### PR TITLE
fixes some formatting and grammar in the Docker log collection docs

### PR DESCRIPTION
**Pull request by le-he-hoo**
### What does this PR do?

Fixes some formatting and some grammar in the Docker log collection documentation

### Motivation

Saw some things to fix in the doc.  For instance, a missing space here - https://cl.ly/0J3L3T153U2r

### CODE REVIEW

File: content/logs/log_collection/docker.md [hunk 1]
```diff
@@ -26,30 +26,30 @@ Two installations are possible:
 - on the host where the Agent is external to the Docker environment
 - or by deploying its containerized version in the Docker environment
 
-You can then choose to collect all the logs from all your environment container or to filter by container image name or container label to cherry pick what logs should be collected.
+You can then choose to collect all the logs from all your environment's containers or to filter by container image name or container label to cherry pick what logs should be collected.
```
Comment (cswatt) [inline content/logs/log_collection/docker.md]: You can then choose to collect all the logs from all your environment's containers, or to filter by container image name or container label to cherry pick what logs should be collected.

File: content/logs/log_collection/docker.md [hunk 2]
```diff
@@ -26,30 +26,30 @@ Two installations are possible:
...
 ## Setup
 ### Option 1: Host installation
 
 Install the [latest version of the Agent 6][1] on your host.
 
-The Agent can both collect logs from [files on the host][2] or from [container stdout/stderr](#configuration-file-example).
+The Agent can collect logs from [files on the host][2] or from [container stdout/stderr](#configuration-file-example).
 
-To collect logs from all your container without filtering by image or label add the following at the end of `docker.d/conf.yaml` in your agent's `conf.d` directory:
+To collect logs from all your containers without filtering by image or label add the following at the end of `docker.d/conf.yaml` in your agent's `conf.d` directory:
```
Comment (cswatt) [inline content/logs/log_collection/docker.md]: To collect logs from all your containers without filtering by image or label, add the following at the end of `docker.d/conf.yaml` in your agent's `conf.d` directory:

File: content/logs/log_collection/docker.md [hunk 3]
```diff
@@ -26,30 +26,30 @@ Two installations are possible:
...
 
-**Important note**: Integration pipelines and processors will not be installed automatically as the source tag is not set. The integrations setup is described below and it automatically installs integration pipelines that parse your logs and extract all the relevant information from them.
+**Important note**: Integration pipelines and processors will not be installed automatically as the source tag is not set. The integration setup is described below and it automatically installs integration pipelines that parse your logs and extract all the relevant information from them.
 
 ### Option 2: Container installation
 
 As explained above, the Agent also has a [containerized][3] installation.
 
-First let’s create one directories on the host that we will later mount on the containerized Agent:
+First, let’s create one directories on the host that we will later mount on the containerized Agent:
```
Comment (cswatt) [inline content/logs/log_collection/docker.md]: I think you mean "directory" — singular

File: content/logs/log_collection/docker.md [hunk 4]
```diff
@@ -112,7 +112,7 @@ logs:
 ```
 
 When filtering on the container image, both exact container image name or short names are supported.
-Suppose you have one container running `library/httpd:latest` the following filtering match this image name:
+If you have one container running `library/httpd:latest`, the following filtering match this image name:
```
Comment (cswatt) [inline content/logs/log_collection/docker.md]: If you have one container running `library/httpd:latest`, the following filtering match**es** this image name:

File: content/logs/log_collection/docker.md [hunk 5]
```diff
@@ -125,7 +125,7 @@ For more examples of configuration files or Agent capabilities (such as filterin
 
 Since version 6.2 of the Datadog Agent, you can configure log collection directly in the container labels.
 
-Autodiscovery expects labels to look like these format, depending on the file type:
+Autodiscovery expects labels to look like this format, depending on the file type:
```
Comment (cswatt) [inline content/logs/log_collection/docker.md]: Autodiscovery expects labels to follow this format, depending on the file type:

Comment (le-he-hoo) [reply content/logs/log_collection/docker.md]: @cswatt  Thanks for the comments!
Please feel free to make any further changes that are needed.

**Pull request merged**